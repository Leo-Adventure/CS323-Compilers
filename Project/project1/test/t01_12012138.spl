int t01_12012138(int a, int b, int c) 
{
  int d = 0
  if(d < a)
    d = a
  
  if(d > b){
    d = b;
  }
  else{
    d = c;
  }
  return d
  }
