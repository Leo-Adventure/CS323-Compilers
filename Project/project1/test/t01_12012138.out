Error type B at Line 3: Missing semicolon ';'
Error type B at Line 5: Missing semicolon ';'
Error type B at Line 13: Missing semicolon ';'
